types/scoped-http-client/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "unified-signatures": false
```

Comment:
This file should just `"extends": "dtslint/dt.json"`, you can disable it in source code like `tslint:disable-next-line unified-signatures`

Reply:
Thank you! Hopefully the file is now good after my last commit.